### PR TITLE
Add node size limit for MPT note

### PR DESCRIPTION
**Pull request by uneet7**
## Fixes
[#1085](https://github.com/0chain/0chain/issues/1085)
## Tests
Tasks to complete before merging PR:
- [ ]  Ensure system tests are passing. If not [Run them manually](https://github.com/0chain/0chain/actions/workflows/system_tests.yml) to check for any regressions :clipboard:
- [ ]  Do any new system tests need added to test this change? do any existing system tests need updated? If so create a PR at [0chain/system_test](https://github.com/0chain/system_test)
- [ ]  Merge your system tests PR to master AFTER merging this PR

## Associated PRs (Link as appropriate):
- blobber:
- gosdk:
- system_test:
- zboxcli:
- zwalletcli:
- Other: ...

### CODE REVIEW

File: code/go/0chain.net/core/util/markle_patricia_trie_interface.go [hunk 1]
```diff
@@ -91,12 +91,13 @@ func GetPruneStats(ctx context.Context) *PruneStats {
...
-	PruneStateDelete    = "deleting"
-	PruneStateCommplete = "completed"
-	PruneStateAbandoned = "abandoned"
+	PruneStateStart         = "started"
+	PruneStateUpdate        = "updating"
+	PruneStateSynch         = "synching"
+	PruneStateDelete        = "deleting"
+	PruneStateCommplete     = "completed"
+	PruneStateAbandoned     = "abandoned"
+	MPTMaxAllowableNodeSize = 1000000 // 1 MB
```
Comment (peterlimg) [inline code/go/0chain.net/core/util/markle_patricia_trie_interface.go]: We'd better put this const in another place that separate it from the PruneState, so perhaps put it in the head of the file above the `ErrIteratingChildNodes`.

**Review by peterlimg — CHANGES_REQUESTED** [review]

File: code/go/0chain.net/core/util/markle_patricia_trie_interface.go [hunk 2]
```diff
@@ -7,6 +7,11 @@ import (
 	"time"
 )
 
+// MPTMaxAllowableNodeSize - Maximum allowable size of MPT node
+const (
+	MPTMaxAllowableNodeSize = 1000000 // 1 MB
```
Comment (peterlimg) [inline code/go/0chain.net/core/util/markle_patricia_trie_interface.go]: Let's use 1024*1024 for 1MB

**Review by peterlimg — CHANGES_REQUESTED** [review]

**Review by dabasov — APPROVED** [review]
lgtm

**Comment by uneet7** [comment]
Closing the PR, because the piece is moved into the `common` repo

**Pull request closed without merging**